testsuite/elytron/server-provisioning.xml

```diff
@@ -19,7 +19,7 @@
   ~ Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
   ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
   -->
-<server-provisioning xmlns="urn:wildfly:server-provisioning:1.0" copy-module-artifacts="false">
```

Comment:
isn't this just workaround ? I think the best practice is to avoid modules copy. cc @ctomc